graph_generator/config/autonomous_vehicle/mapping.yaml

```diff
@@ -88,10 +88,158 @@ nodes:
             affect_publish:
               topic: road_data
               drop: 5
+
+      - topic: processed_camera_data5
+        valid_range: [500, 600]
+        watchdog: 7
+        lost_input_callback:
+          fault:
+            affect_publish:
+              topic: road_data
+              drop: 5
+              
+      - topic: processed_camera_data8
+        valid_range: [800, 900]
+        watchdog: 7
+        lost_input_callback:
+          fault:
+            affect_publish:
+              topic: road_data
+              drop: 5
+
     loop:
       period: 20
       callback:
         publish:
           - topic: road_data
             value_range: [20, 30]
             delay_range: [1, 2]
+
+  - name: localization
+    subscribe:
+      - topic: gps1
+        valid_range: [0, 10]
+        watchdog: 10
```

Comment:
this is never satisfied. gps1 and gps2 are published periodically every 20 time units.